### PR TITLE
Global improvements of the docs website

### PR DESCRIPTION
**Pull request by Odonno**
I tried to improve the global design of the website by mostly fixing typos and realigning HTML elements.

Here is a glimpse of the improvements made:

image

### CODE REVIEW

File: src/pages/features/doc-strings.mdx [hunk 1]
```diff
@@ -19,27 +19,27 @@ import Layout from '../../layouts/Layout.astro'
...
-    DocStrings value is passed at last parmeters in your step callback : 
+    DocStrings value is passed as the last parameter in your step callback: 
 
     ~~~typescript
     describeFeature(feature, (f) => {
         f.Scenario(`DocStrings example`, (s) => {
-            s.Given(`I use vitest-cucumber`, (ctx, docStrings: string) => {
-                expect(docStrings.split('\n')).toEqual([,
+            s.Given(`I use vitest-cucumber`, (ctx, description: string) => {
+                expect(description.split('\n')).toEqual([,
```
Comment (amiceli) [inline src/pages/features/doc-strings.mdx:28]: I prefer `docStrings` instead of `description`.

File: src/pages/features/gherkin.mdx [hunk 2]
```diff
@@ -3,15 +3,15 @@ import Layout from '../../layouts/Layout.astro'
 <Layout title="Gherkin">
 	# Gherkin validation
 
-    **vitest-cucumber** use its own feature file parser and validate it before 
-    run unit tests.
+    **vitest-cucumber** uses its own feature file parser and validate it before 
+    running unit tests.
 
-    And checl you test all scenario, steps etc.
+    And check ths tests for all scenario, steps, etc...
```
Comment (amiceli) [inline src/pages/features/gherkin.mdx:9]: Not `...` with etc.

**Comment by amiceli** [comment]
The rest is Ok, I will merge it.

File: src/pages/features/gherkin.mdx (hunk 2 above)
Comment (Odonno) [reply src/pages/features/gherkin.mdx:9]: That was my preference but it seems to be `etc.` indeed.

**Comment by Odonno** [comment]
Should be ok now.

**Comment by amiceli** [comment]
Yes, I merge it.

**Pull request merged**